### DISCUSSION TITLE
ORAXEN NOT WORK IN 1.20.6 (work in 1.20.4 & 1.21)

### DISCUSSION THREAD

**Discussion by Astrotix** [DerFrZocker/Custom-Ore-Generator#38] [Q&A]
Hey,

This doesn't work in 1.20.6 but working in 1.20.4 & 1.21!. ORAXEN 1.181 The generation only look NOTE_BLOCK. Possible to patch it ? Thanks !

value:
  ==: CustomOreGenerator#OreConfig
  name: opale
  material: NOTE_BLOCK
  ore-generator: VANILLA_MINABLE_GENERATOR
  block-selector: COUNT_RANGE
  activated: true
  generated-all: false
  biomes:
  - SNOWY_BEACH
  - FOREST
  - GROVE
  - SAVANNA_PLATEAU
  - WOODED_BADLANDS
  - SNOWY_SLOPES
  - SUNFLOWER_PLAINS
  - DESERT
  - SAVANNA
  - WINDSWEPT_SAVANNA
  - SNOWY_TAIGA
  - MUSHROOM_FIELDS
  - BEACH
  - JAGGED_PEAKS
  - LUSH_CAVES
  - OLD_GROWTH_PINE_TAIGA
  - BAMBOO_JUNGLE
  - BADLANDS
  - TAIGA
  - FROZEN_OCEAN
  - WINDSWEPT_FOREST
  - ERODED_BADLANDS
  - DEEP_FROZEN_OCEAN
  - DEEP_OCEAN
  - FROZEN_PEAKS
  - FROZEN_RIVER
  - STONY_PEAKS
  - LUKEWARM_OCEAN
  - OLD_GROWTH_SPRUCE_TAIGA
  - DEEP_COLD_OCEAN
  - SWAMP
  - PLAINS
  - WINDSWEPT_GRAVELLY_HILLS
  - DRIPSTONE_CAVES
  - OLD_GROWTH_BIRCH_FOREST
  - COLD_OCEAN
  - ICE_SPIKES
  - BIRCH_FOREST
  - RIVER
  - OCEAN
  - FLOWER_FOREST
  - SPARSE_JUNGLE
  - STONY_SHORE
  - MANGROVE_SWAMP
  - MEADOW
  - WARM_OCEAN
  - WINDSWEPT_HILLS
  - SNOWY_PLAINS
  - DARK_FOREST
  - DEEP_LUKEWARM_OCEAN
  - JUNGLE
  replace-materials:
  - STONE
  select-materials:
  - STONE
  custom-data:
    ORAXEN: opale_ore
  ore-generator-settings:
    ==: CustomOreGenerator#OreSettingsContainer
    ore-settings:
      VEIN_SIZE: 2.0
  block-selector-settings:
    ==: CustomOreGenerator#OreSettingsContainer
    ore-settings:
      MINIMUM_HEIGHT: 0.0
      VEINS_PER_CHUNK: 6.0
      HEIGHT_RANGE: 10.0
![2024-08-24 19_31_12-RE_ Mise en place EPC - Message (HTML)](https://github.com/user-attachments/assets/e5d7255a-26fd-48e4-b4c4-8b17c78823a4)

**Comment by Storm-Dev6** [comment]
Problem off the generators its been:
```generated-all: false``` 

Make sure you hv a generator on true ```mode```

Best Regard!
